mk/xe-linux-distribution-test

```diff
@@ -66,27 +77,36 @@ do_test "testcases/rhel-5beta"           "rhel 5 0beta"
 do_test "testcases/rhel-5"               "rhel 5 0"
 do_test "testcases/rhel-5u1beta"         "rhel 5 1beta"
 do_test "testcases/rhel-5u1"             "rhel 5 1"
-do_test "testcases/fc3"                  "fedora 3 0"
 do_test "testcases/sles-9"               "sles 9 0"
 do_test "testcases/sles-9sp3"            "sles 9 3"
 do_test "testcases/sles-10sp2"           "sles 10 2"
 do_test "testcases/sles-11"              "sles 11 0"
-do_test "testcases/ddk-0.5.6-2991c"      "xe-ddk 0.5.6 2991c"
-do_test "testcases/sdk-0.5.6-2991c"      "xe-sdk 0.5.6 2991c"
 do_test "testcases/centos4"              "centos 4 0"
 do_test "testcases/centos5"              "centos 5 0"
+do_test "testcases/centos8.1"            "centos 8 1"
+do_test "testcases/centos8.3"            "centos 8 3"
 do_test "testcases/lsb-ubuntu-6.10"      "ubuntu 6 10"
 do_test "testcases/lsb-ubuntu-7.04"      "ubuntu 7 04"
 do_test "testcases/oracle-5"             "oracle 5 0"
 do_test "testcases/oracle-5u1"           "oracle 5 1"
 do_test "testcases/lsb-coreos-367.1.0"   "coreos 367 1.0"
-do_test "testcases/sle-15"               "sles 15 0"
-do_test "testcases/sle-15sp1"            "sles 15 1"
-do_test "testcases/centos7"              "centos 7 0"
-do_test "testcases/centos8.1"            "centos 8 1"
-do_test "testcases/centos8.3"            "centos 8 3"
-do_test "testcases/centos-stream-8"      "centos 8 0"
-do_test "testcases/rocky-8.3"            "rocky 8 3"
-do_test "testcases/almalinux-8.3"        "almalinux 8 3"
-do_test "testcases/slackware-15.0"       "slackware 15 0"
+# Newly supported distros
```

Comment:
Please add the newly added test cases into the previous one that not EOL, the work "Newly supported distros" may cause misunderstanding, actually only add new test cases, not to newly support the new distros